.changeset/clever-eagles-try.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/website': patch
+---
+
+Add Keystone 5 vs Next guide to site
```

Comment:
```suggestion
Added a Keystone 5 vs Next guide to site.
```

The comments we leave in changesets are used to generate our `CHANGELOG.md` contents. For consistency we want these changelogs to be written in **past tense** and **complete sentences** (e.g. capital letters, full stops, etc). This is somewhat motivated by pedantry on my part, but I think it makes for a better reading experience when people go to find out what has changed.